### PR TITLE
NDI-8: Restore UI spinner styles

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
# Description
- This PR restores the style for `.ui-spinner-input` to make it same as Vanilla CiviCRM again on search result pages.

# Screenshots
**Before**
image
**After**
image

# Technical Details
`.ui-spinner-input` is get overridden by Bootstrap and Shoreditch style though, it should be looking as it is without any change. So, We are again overriding all the styles back as per Vanilla CiviCRM

# Comments
- Stylelint tested
- BackstopJS tested

### CODE REVIEW

**Comment by AkA84** [comment]
@mukeshcompucorp 

1. The PR is about a spinner, but in the screenshot I don't see any, there is a dropdown that is highlighted
2. Why is something in the bootstrap side of the theme affecting things on core screens?

**Comment by mukeshcompucorp** [comment]
@AkA84
1. in Vanilla CiviCRM they call this control spinner, an input with arrows (see markup screenshot)
image

2. As this is `input[type='text']` it gets default style we did for all inputs which should not be for this specific input

File: scss/civicrm/search/pages/_results.scss [hunk 1]
```diff
@@ -58,14 +60,26 @@
...
       margin-top: 7px;
     }
+  }
 
-    + .float-right {
-      float: none !important;
-      padding: 4px 10px 10px !important;
-      text-align: right;
-    }
+  .float-right {
```
Comment (AkA84) [inline scss/civicrm/search/pages/_results.scss]: Can you show me what this is for?

File: scss/civicrm/search/pages/_results.scss [hunk 2]
```diff
@@ -58,14 +60,26 @@
...
+    float: none !important;
+    padding: 10px 20px 20px !important;
+    text-align: right;
+  }
+
+  .ui-spinner-input {
```
Comment (AkA84) [inline scss/civicrm/search/pages/_results.scss]: but this should be done in any page, right? Shouldn't this be in a more generic partial?

Comment (mukeshcompucorp) [reply scss/civicrm/search/pages/_results.scss]: Yes, that makes sense, I will move

File: scss/civicrm/search/pages/_results.scss (hunk 1 above)
Comment (mukeshcompucorp) [reply scss/civicrm/search/pages/_results.scss]: See below screenshot
image

**Pull request merged**